gcp/rakefiles/test.rake

```diff
@@ -0,0 +1,23 @@
+desc '[TEST] Run Locust swarm against Preferences service in current cluster'
+task :test_preferences => [:set_vars] do
+  sh "#{@exekube_cmd} rake xk[' \
+    xk down live/dev/locust && \
+    sleep 30 && \
+    TF_VAR_locust_target_host=http://preferences.$TF_VAR_domain_name \
+    TF_VAR_locust_script=preferences.py \
+    xk up live/dev/locust',skip_infra,skip_secret_mgmt]"
```

Comment:
Do we want to leave Locust running after the test completes? I think it would be better to clean up, especially since we're going to tear everything down before the next test run (which we should continue to do).

Reply:
It turns out we don't need to destroy Locust module to reconfigure it. Terraform detects template changes and modifies Helm release if we provide TF_VARs with different values. I could add `destroy_module` command to CI pipeline, but module will be destroyed with the cluster anyway (it does not leave any leftover resources behind).

Reply:
Nevermind the good news, there is a pretty good chance that Locust master configuration becomes messed up if we do not destroy module before.

1. I don't have strong opinion on this. If we include tests to `rake` / `rake deploy` workflow, it would add around 5 minutes of run time, which may be not very convenient for `dev` environment, if you just want to quickly spin everything up to debug your module. We can probably just copy tests from AWS side, and include them, but it will require `wait_for` code and maybe something else.
2. Added.
3. Agree, added.

Reply:
I agree 5 minutes is a long time.

I think we can table this for now. `gcp-*` jobs will run a superset of smoke tests as part of CI. We can discuss other options later.